jbmc/src/java_bytecode/java_types.h

```diff
@@ -155,6 +155,19 @@ class java_class_typet:public class_typet
       class_typet::get_component(component_name));
   }
 
+  using static_membert = componentt;
+  using static_memberst = std::vector<componentt>;
+
+  const static_memberst &static_members() const
+  {
+    return (const static_memberst &)class_typet::static_members();
```

Comment:
`static_cast` preferred

Reply:
This doesn't work here, it's an unrelated type.